### PR TITLE
feat: add-eslint

### PR DESCRIPTION
**Pull request by mrnerdhair**
Supercedes #453. It's the same thing, done by @NeOMakinG, but rebased for easier review. Needed its own PR since I rewrote the history.

### CODE REVIEW

File: integration/src/secret/secret.ts [hunk 1]
```diff
@@ -51,7 +51,7 @@ export function secretTests(get: () => { wallet: core.HDWallet; info: core.HDWal
             path: core.bip32ToAddressNList("m/44'/529'/0'/0/0"),
             coin: "Secret",
           })
-        );
+        ).toMatchInlineSnapshot();
```
Comment (NeOMakinG) [inline integration/src/secret/secret.ts:54]: I think we should generate it!

**Comment by NeOMakinG** [comment]
@mrnerdhair just a little thing that I'll try to push today, then let's open it for review?

File: integration/src/secret/secret.ts (hunk 1 above)
Comment (NeOMakinG) [reply integration/src/secret/secret.ts:54]: Ok I get it, it doesn't return anything, we don't care then

**Comment by mrnerdhair** [comment]
Rebased, updated, and ready for review!

**Review by cjthompson — APPROVED** [review]

**Pull request merged**